float64 x_alpha
float64 x_alpha_dot
float64 x_vel
float64 x_theta
float64 x_theta_dot
float64 u_vel
float64 u_w